scalafix-tests/input/src/main/scala/tests/ExplicitResultTypesRefinement.scala

```diff
@@ -27,10 +27,11 @@ object ExplicitResultTypesRefinement {
   def tparam[T <: CharSequence](e: T) = new Serializable {
     val results: List[Int] = List(e.length())
   }
-  val access = new Serializable {
-    private val results: List[Int] = List.empty
-    protected val results2: List[Int] = List.empty
-  }
+  //todo: in 2.13, the type found is AnyRef
```

Comment:
It's usually better to open an issue in my experience and link to from here instead of adding a todo comment.

Reply:
I will create all corresponding issues.